### DISCUSSION TITLE
Simultaneous LEO (kinematic) and GNSS orbit determination

### DISCUSSION THREAD

**Discussion by laplace76** [groops-devs/groops#259] [Q&A]
Dear Developers,

I would like to estimate GNSS orbit/clocks/biases and LEO kinematic orbit/clock in the same least squares adjustment. Therefore, I have introduced sentinel 1 as an additional receiver - type lowOrbiter - and added a specific ionoVTEC parametrization (after having prepared the a priori leo orbit and attitude files). The issue I have now is the different sampling rate (sentinel sampling rate is 10s while ground receiver's is 30s) and initial epochs. How can I handle this?

Kind Regards

**Comment by student-account-s** [comment]
Dear Author,
I’d like to ask whether you’ve tried using the existing GROOPS project code to implement GNSS/LEO joint orbit determination.
Best regards,
Wmj
Beijing Normal University
